src/Components/Header.css

```diff
@@ -0,0 +1,62 @@
+.header {
+  width: 100%;
+  display: flex;
+  height: 5.938rem;
+  margin: 0 0 1.375rem;
+  padding: 1.438rem 0 1.688rem 5%;
+  background-color: #fff;
+}
```

Comment:
- [ ] Kindly make your header look like the template one by aligning everything properly, use flex instead of paddings to align your items vertically. Check here the deference. 

Your header | The template header
----|----
![image](https://user-images.githubusercontent.com/65068771/155844873-b5dd3573-d5e4-4af5-9ceb-50af99be7432.png) | ![image](https://user-images.githubusercontent.com/65068771/155844912-7d70ed21-1233-4dae-8fc5-e12b14ba5ac0.png)

- [ ] Also put the icon at the left of your header as the requirements say that Features that haven’t been implemented yet should be added as a placeholder.

Reply:
Thanks @andersonlebon I've enough to amend it. That's cool